Datasets/data-connect-dataset-plannertasks.md

```diff
@@ -0,0 +1,121 @@
+
+The PlannerTasks_v0 dataset enables you to identify all the Planner tasks assigned to any of the users of a tenant that are secured by Azure Active Directory (Azure AD) in Microsoft 365. User accounts can be on Microsoft 365 or a Microsoft account (Hotmail.com, Live.com, MSN.com, Outlook.com, and Passport.com). 
+
+## Scenarios
+
+The following are business scenarios that you can answer with this dataset:
+
+- Analyze the number of tasks that were created by all users. You can also analyze how many of these tasks got completed before the assigned due dates, and how many got completed after the assigned due dates.
+- Identify stale tasks for notifying respective users to complete their tasks before the due date.
+- Identify users having incomplete tasks beyond a particular threshold.
+- Identify tasks assigned to users where the status remains unchanged beyond a particular threshold.
+
+## Questions
+
+The following are examples of questions that you can answer with this dataset:
+
+- How many tasks were created by all users of a tenant?
+- How many tasks were completed before the assigned due date by all users of a tenant?
+- How many tasks were completed after the assigned due date by all users of a tenant?
+- How many tasks were assigned to all users during a time range? 
+- How many average tasks are there inside a plan? 
+- What percentage of Tasks got completed during a time range? 
+- What is the average time it takes for any Task to move from InProgress state to Completed state? 
+- Which/How many Tasks have surpassed the Due Date but still haven’t been started? 
+
+## Joining with other datasets
+
+The PlannerTasks_v0 dataset can be joined with relevant users, and other category datasets.
+
+## Definitions
+
+- **Task:** Represents a Planner task in Microsoft 365. A Planner task is contained in a plan and can be assigned to a bucket in a plan. 
+- **Plan:** Plans are the containers of tasks. A plan can be owned by a group and contains a collection of plannerTasks.
+
+
+## Schema
+
+| Name  | Type  |  Description  |  FilterOptions  |  FilterType  |
+| ----------- | ----------- | ----------- | ----------- | ----------- |
+| planId | String | Plan ID to which the task belongs. | No | None |
+| bucketId | String | Bucket ID to which the task belongs. The bucket needs to be in the plan that the task is in. It is 28 characters long and case-sensitive. | No | None |
+| title | String | Title of the task. | No | None |
+| orderHint | String | Hint used to order items of this type in a list view. | No | None |
+| assigneePriority | String | Hint used to order items of this type in a list view. | No | None |
+| percentComplete | Int32 | Percentage of task completion. When set to 100, the task is considered completed. | No | None |
+| startDateTime | Datetime | Date and time at which the task starts. It is in UTC format. | No | None
+| createdDateTime | Datetime | Date and time at which the task is created. It is in UTC format. | 1 | Date |
```

Comment:
Please make the FilterOptions=Yes for the createdDateTime instead of "1"

Reply:
Done